cherry_picker/test.py

```diff
@@ -1,7 +1,19 @@
-from . import cherry_picker
+from unittest import mock
 
+from .cherry_picker import get_base_branch, get_current_branch
+
+def test_get_base_branch():
+    cherry_pick_branch = 'backport-afc23f4-2.7'
+    result = get_base_branch(cherry_pick_branch)
+    assert result == '2.7'
+
+def test_get_base_branch_without_dash():
+    cherry_pick_branch ='master'
+    result = get_base_branch(cherry_pick_branch)
+    assert result == 'master'
+
+@mock.patch('subprocess.check_output')
+def test_get_current_branch(subprocess_check_output):
+    subprocess_check_output.return_value = b'master'
+    assert get_current_branch() == 'master'
 
-def test_sorted_branch():
```

Comment:
It would be handy to keep this test, either by splitting it out to a help function that's called by the property implementation, or else by testing `cherry_picker.CherryPicker.sorted_branches.fget` directly on a dummy object with a suitable `branches` attribute.